package.json

```diff
@@ -19,7 +19,7 @@
     "chai": "^4.2.0",
     "eclint": "^2.8.1",
     "eslint": "^5.13.0",
-    "eslint-config-canonical": "^17.3.4",
+    "eslint-config-canonical": "^18.1.1",
```

Comment:
This dependency was causing an error because in this commit https://github.com/benmosher/eslint-plugin-import/commit/1a3a12869651bec54d567af8dc0e7cb2a801d41f they added a validation for a rule they were using.